src/pages/accounts/AccountDetails.tsx

```diff
@@ -1,451 +1,321 @@
-import { AccountsUrl } from '../../services/ApiUrls'
-import { Tags } from '../../components/Tags'
-import { CustomAppBar } from '../../components/CustomAppBar'
-import { FaPlus, FaStar } from 'react-icons/fa'
-import FormateTime from '../../components/FormateTime'
-import { Label } from '../../components/Label'
-import { AntSwitch } from '../../styles/CssStyled'
-import MyContext, { useMyContext } from '../../context/Context'
-
-export const formatDate = (dateString: any) => {
-    const options: Intl.DateTimeFormatOptions = { year: 'numeric', month: 'long', day: 'numeric' }
-    return new Date(dateString).toLocaleDateString(undefined, options)
-}
-type response = {
-    created_by: {
-        email: string;
-        id: string;
-        profile_pic: string;
-    };
-    user_details: {
-        email: string;
-        id: string;
-        profile_pic: string;
-    };
-    org: { name: string };
-    lead: { title: string };
-    account_attachment: [];
-    assigned_to: Array<{ user_details: { email: string; id: string; profile_pic: string | null; } }>;
-    billing_address_line: string;
-    billing_city: string;
-    billing_country: string;
-    billing_state: string;
-    billing_postcode: string;
-    billing_street: string;
-    contact_name: string;
+interface Account {
+  name: string;
+  account_name: string;
+  deals:string;
+  account_obj: {
     name: string;
-
-    created_at: string;
-    created_on: string;
-    created_on_arrow: string;
-    date_of_birth: string;
-    title: string;
-    first_name: string;
-    last_name: string;
-    account_name: string;
-    phone: string;
-    email: string;
-    lead_attachment: string;
-    opportunity_amount: string;
     website: string;
-    description: string;
-    contacts: string;
-    status: string;
-    source: string;
-    address_line: string;
-    street: string;
-    city: string;
-    state: string;
-    postcode: string;
-    country: string;
-    tags: [];
-    company: string;
-    probability: string;
     industry: string;
-    skype_ID: string;
-    file: string;
-
-    close_date: string;
-    organization: string;
-    created_from_site: boolean;
-    id: string;
-    teams: [];
-    leads: string;
-
-};
-export const AccountDetails = (props: any) => {
-    const { state } = useLocation()
-    console.log(state)
-    const navigate = useNavigate()
-    const { userRole , profileId, userId} = useMyContext();
+    org: {
+      name:string;
+  
+    };
+    assigned_to: [
+    {
+      user_details: {
+        email: string;
+      };
+    }
+  ];};
+  
+  contacts: [
+    {
+      first_name: string;
+      last_name: string;
+      primary_email: string;
+      mobile_number: string;
+      department: string;
+      address: {
+        street: string;
+        city: string;
+        state: string;
+        postcode: string;
+        country: string;
+      };
+    }
+  ];
+  website: string;
+  
+ 
+  description: string;
```

Comment:
if description, moves up under "account_obj" , and in div we access it via `account.account_obj.description` then everything is working fine.
I already merged the patch in the backend to the develop branch.